modules/cnv-importing-vmware-vm.adoc

```diff
@@ -1,67 +1,77 @@
 // Module included in the following assemblies:
 //
-// * cnv/cnv_users_guide/cnv-create-vms.adoc
+// * cnv/cnv_users_guide/cnv-importing-vmware-vm.adoc
+[id='Importing_vmware_vm_or_template_{context}']
+= Importing a VMware virtual machine or template
 
-= Importing the VMware virtual machine or template
+You can import a VMware virtual machine or template using the virtual machine wizard.
 
 .Prerequisites
 
-* The VMware virtual machine is powered off.
+* You must create a VDDK image, push it to a private container repository, and add it to the `v2v-vmware` Configmap.
+* You must power off the VMware virtual machine before importing it.
 
 .Procedure
 
-. Click *Workloads* -> *Virtual Machines* from the side menu.
-. Click *Create Virtual Machine* and select *Create with Wizard*.
-. Perform the following procedure in the *Basic Settings* screen:
-.. Click *Provision Source* and select *Import*.
-.. Click *Provider* and select *VMware*.
-.. Click *vCenter Instance* and select *Connect to New Instance* or a saved vCenter instance.
-+
-* If you select *Connect to New Instance*, provide values in the following fields and click *Check and Save*:
+. In the {CNVProductName} web console, click *Workloads* -> *Virtual Machines*.
+. Click *Create Virtual Machine* and select *Import with Wizard*.
+. In the *General* screen, perform the following steps:
+.. Select *VMware* from the *Provider* list.
+.. Select *Connect to New Instance* or a saved vCenter instance from the *vCenter instance* list.
+
+** If you select *Connect to New Instance*, fill in the *vCenter hostname*, *Username*, and *Password*.
+** If you select a saved vCenter instance, the  wizard connects to the vCenter instance using the saved credentials.
 
-** *vCenter Hostname*
-** *vCenter User Name*
-** *vCenter Password*
+.. Select a virtual machine or a template to import from the *VM or Template to Import* list.
+.. Select an operating system.
+.. Select an existing flavor or *Custom* from the *Flavor* list.
 +
-The wizard connects to the vCenter instance and saves the credentials.
+If you select *Custom*, specify the *Memory (GB)* and the *CPUs*.
+.. Select a *Workload Profile*.
+.. If the virtual machine name is already being used by another virtual machine in the namespace, update the name.
+.. Click *Next*.
+
+. In the *Networking* screen, perform the following steps:
+
+.. Click the Options menu {kebab} of a network interface and select *Edit*.
+.. Enter a valid network interface name.
 +
-If you clear the *Save as New vCenter Instance* check box and click *Check*, the wizard connects to the vCenter instance without saving the credentials.
+The name can contain lower-case letters (`a-z`), numbers (`0-9`), and hyphens (`-`), up to a maximum of 253 characters. The first and last characters must be alphanumeric. The name must not contain upper-case letters, spaces, periods (`.`), or special characters.
```

Comment:
Should be "lowercase" and "uppercase" (according to IBM style guide)

Same elsewhere in the PR

Reply:
I copied it from a couple other modules (wizard fields). In that case, I'll correct it there as well.